pkg/query-service/app/cloudintegrations/services/models.go

```diff
@@ -0,0 +1,91 @@
+package services
```

Comment:
why renamed from model to models ?

Reply:
also see if you can move this to types package

Reply:
Scoped usecase for these structs, plus some of them might clash due to similar names. I'll need clear guidelines on what to be in Types, and what to keep scoped.